core-plugins/src/main/java/co/cask/hydrator/plugin/batch/action/OracleExportAction.java

```diff
@@ -139,19 +136,17 @@ private String buildOracleExportCommand() {
   }
 
   private String getColSeparator(String format) {
-    String columnSeparator = new String();
     switch (SeparatorFormat.valueOf(format)) {
-      case csv:
-        columnSeparator = ",";
-        break;
-      case tsv:
-        columnSeparator = "  ";
-        break;
-      case psv:
-        columnSeparator = "|";
-        break;
+      case CSV:
+        return ",";
+      case TSV:
+        return "  ";
+      case PSV:
+        return "|";
+      default:
```

Comment:
Default is included to check for the illegal formats which gets escaped from the validate method in Config classes.